resources/deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
```

Comment:
Is this deployment file used for all kinds of deployments or just Keycloak deployments? Deployment files can vary alot depending on the deployed system

Reply:
Deployment.yaml is used to create a Mikikube deployment, service.yaml is used to create a Minikube service, pod.yaml is used to create a Minikube pod.